src/index.html

```diff
@@ -6,10 +6,9 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://cdn.ng-buch.de">
 </head>
 <body>
-  <bm-root>
-    <div class="loader">Loading ...</div>
-  </bm-root>
+  <bm-root></bm-root>
```

Comment:
Kein Loader mehr?

Reply:
Das war wohl ein Versehen.

```suggestion
  <bm-root>
    <div class="loader">Loading ...</div>
  </bm-root>
```